draft-ietf-rats-corim.md

```diff
@@ -1123,26 +1123,38 @@ the object relates to the subject.
 #### Device Identity Triple {#sec-comid-triple-identity}
 
 A Device Identity triple relates one or more cryptographic keys to a device.
-The subject of an identity triple uses an instance or class identifier to refer
-to a device, and a cryptographic key is the object. The predicate asserts that
-at least one of the keys, as defined by `$crypto-key-type-choice`, authenticates the Attester.
+The triple subject is an identity used to name a device.
+The triple object is an array of cryptographic key(s) used to authenticate the device.
+The predicate asserts that any of the key(s) (found in the triple object) can be used to authenticate the device.
+
+If contained in a CoRIM, this triple endorses that the key(s) were provisioned to the Target Environment.
+
+If contained in Evidence, this triple asserts that the key(s) are present in the Target Environment.
+
+The Verifier SHOULD attempt to verify the status and validity of the key(s).
 
 Depending on which `$crypto-key-type-choice` form is supplied, the Verifier may
-need to take different steps to locate a suitable credential.
+need to take different steps to locate or verify the key.
 
 ~~~ cddl
 {::include cddl/identity-triple-record.cddl}
 ~~~
 
 #### Attestation Keys Triple {#sec-comid-triple-attest-key}
 
-An Attestation Keys triple relates one or more cryptographic keys to an
-Attesting Environment. The attest key triple subject is an Attesting
-Environment whose object is a cryptographic key. The predicate asserts that the
-Attesting Environment integrity protects Evidence using at least one of the keys.
+An Attestation Keys triple relates one or more cryptographic keys to an Attesting Environment.
+The attest key triple subject identifies an Attesting Environment
+whose object is an array of cryptographic key(s) used to attest Evidence.
+The predicate asserts that any of the key(s) (found in the triple object) can be used to attest Evidence.
+
+If contained in a CoRIM, this triple endorses that the key(s) were provisioned to the Attesting Environment.
+
+If contained in Evidence, this triple asserts that the key(s) are present in the Attesting Environment.
```

Comment:
This sentence should be in the Concise Evidence spec rather than here.

Reply:
> This sentence should be in the Concise Evidence spec rather than here.

When a claim is added to the ACS, the semantics of the claim are asserted by the verifier. This specification should capture these semantics. If another spec defines evidence that is mapped into this schema. The mapping expects the semantics are the same since mapping different semantics can be challenging. Stating it here ensures the other evidence forms are semantically similar/same.